app/models/collection_version.rb

```diff
@@ -40,6 +40,14 @@ def accessioned?
       DepositCollectionJob.perform_later(collection_version)
     end
 
+    after_transition on: :update_metadata do |collection_version, transition|
+      if transition.from == 'new' && transition.to == 'first_draft'
+        FirstDraftCollectionsMailer.with(collection_version: collection_version,
+                                         creator: collection_version.collection.creator)
```

Comment:
I don't see where creator is ever used in the FirstDraftCollectionsMailer. Can this be omitted?

Reply:
it is used in the email.

Reply:
nm - it's not - good catch.  :-)